server.js

```diff
@@ -1,10 +1,26 @@
 const express = require('express')
 const app = express()
-const port = 3001
-app.get('/ping' , (req, res) => {
-    res.send('Hello everyone')
+const mongoose = require('mongoose')
+const port = 3000
+
+const uri = 'mongodb+srv://Surya:sun@asapcluster.3osvgms.mongodb.net/?retryWrites=true&w=majority';
```

Comment:
Remove the hardcoded MongoDB URI and use environment variables instead. This will improve the security of your application. [important]